app/models/user.rb

```diff
@@ -31,7 +31,7 @@ class User < ApplicationRecord
   after_validation :reverse_geocode
 
   after_create :send_welcome_email
-  after_create :subscribe_to_newsletter
+  # after_create :subscribe_to_newsletter
```

Comment:
Once a user is created the `:subscribe_to_newsletter` subscribers to MailChimp. The problem is that each time that I run `seeds.rb` in development, it fails as the user already exists in MailChimp. Any ideas on how can I make `seeds.rb` works even when Mailchimp return a failure? 

Alternatively, how could I run this method in development only?